### ISSUE TITLE
_status.json content should be included in Rest API

### ISSUE THREAD

**Issue by AnalogJ** [dadoonet/fscrawler#798]
**Is your feature request related to a problem? Please describe.**

I started up fscrawler with the `--rest` flag, hoping to make the content of the `_status.json` file available to my webapp. Unfortunately, after looking at the Rest code, it doesn't seem like thats an implemented endpoint. 

**Describe the solution you'd like**

The _status.json content should be included via the Status endpoint `/`, at the very least "lastrun"
 
```
{
  "name" : "lode",
  "lastrun" : "2019-08-23T06:10:00.277",
  "indexed" : 0,
  "deleted" : 0
}```

Thanks!

**Comment by dadoonet** [comment]
Agreed. Is this something you'd like to implement yourself and contribute?

**Comment by dadoonet** [comment]
I think this has been added with #2308
